src/clr-angular/typography/_typography.scss

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
```

Comment:
For the first version of core we are just shipping the components. I removed the core global styles from pulling in the typography and grid until we have time to evaluate how we want core users to adopt those styles.